src/components/fields/AddressField.js

```diff
@@ -68,21 +77,22 @@ const AddressField = (
     return setIsValid(newIsValid);
   }, []);
 
-  const handleChange = useCallback(
-    ({ nativeEvent: { text } }) => {
-      onChange(text);
+  const handleChangeText = useCallback(
+    text => {
       validateAddress(text);
       expandAbbreviatedClipboard();
+      setInputValue(text);
+      onChangeText(text);
     },
-    [expandAbbreviatedClipboard, onChange, validateAddress]
+    [setInputValue, expandAbbreviatedClipboard, onChangeText]
   );
 
   useEffect(() => {
     if (name !== inputValue || name !== address) {
       setInputValue(name);
       validateAddress(address);
     }
-  }, [address, editable, inputValue, name, validateAddress]);
+  }, [address, editable, name, validateAddress]);
```

Comment:
i'm getting lint warnings, `inputValue` is missing

Reply:
Disabled that rule there, I specifically need to not run the effect on each inputValueChange but still need to compare things against the inputValue. If you have ideas how to make it better with this constratints, please let me know 😄

Reply:
sounds fragile. I would do this with ref probably, but i don't mind this as well